app/models/lab_test.rb

```diff
@@ -3,9 +3,33 @@
 class LabTest < ApplicationRecord
   belongs_to :user
   belongs_to :biomarker
-  belongs_to :reference_range
   belongs_to :recordable, polymorphic: true, touch: true
+  belongs_to :reference_range
 
-  validates :value, presence: true, numericality: true
   validates :unit, presence: true
+  validates :value, presence: true,
+                    numericality: {
+                      in: 0..Float::INFINITY
+                    }
+
+  module Status
```

Comment:
I don't think we need a module here. These Statuses are tight to LabTest class, so nested class would be enough.